stix_shifter_modules/vectra/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1216 @@
+{
+  "src_ip": [
```

Comment:
is `src_ip` not associated with any network-traffic objcet? i see dst_ip is mapped to dst_ref for network-traffic.

Reply:
All network-traffic objects are constructed from the nested list in the source data while as src_ip is mapped outside the list. Framework doesn't support referencing src_ip into the nested list as reference is index based. For a particular record there is only a single host which is compromised and this src_ip is mapped to x-ibm-finding.src_ip_ref as well as to x-oca-asset.ip_refs.

Reply:
But ideally that's part of network-traffic object. I also see `src_host` field in the sample results which contains an IP but don't see it in the mapping though. If its one source to many dest that should also be fine.

Reply:
In case of multiple network-traffic objects from a list of corresponding data source dict, 
when we tried to map, it(ipv4-addr object created for src_ip) was added as reference to first network-object only due to stix-shifter framework constraint.
src_host is mapped to x-oca-asset. src_ip is referenced to x-oca-asset.ip_refs from outside src_host , being same.

Reply:
as discussed its fine to reference src_ip to the first network traffic object.

Reply:
Updated mapping to refer src_ip in network traffic object.